test_scripts/Smoke/API/036_UnregisterAppInterface_PositiveCase.lua

```diff
@@ -0,0 +1,49 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: UnregisterAppInterface
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [UnregisterAppInterface] SUCCESS: getting SUCCESS:UnregisterAppInterface()
+--
+-- Description:
+-- Mobile application sends valid UnregisterAppInterface request and gets UnregisterAppInterface "SUCCESS"
+-- response from SDL
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests UnregisterAppInterface
+
+-- Expected:
+-- SDL checks if UnregisterAppInterface is allowed by Policies
+-- SDL sends the BasicCommunication notification to HMI
+-- SDL responds with (resultCode: SUCCESS, success:true) to mobile application
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Local Functions ]]
+local function unregisterAppInterface(self)
+	local cid = self.mobileSession1:SendRPC("UnregisterAppInterface", { })
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppUnregistered", { appID = self.appID, unexpectedDisconnect = false })
```

Comment:
@GetmanetsIrina Currently ```self.appID``` is nill. It should be used ```getHMIAppId()``` function from common module instead.

Reply:
@dboltovskyi , Done in 607c20fb6767f2c4c0e34e662869ef978977e810.